view/frontend/web/js/view/checkout/email.js

```diff
@@ -1,40 +1,61 @@
 define([
     'uiComponent',
     'jquery',
-    'jquery/ui'
+    'jquery/ui',
+    '!domReady'
 ], function(Component) {
     'use strict';
+    // initialize the customerData prior to returning the component
+    var _klaviyoCustomerData = window.customerData;
```

Comment:
what is window.customerData?

Reply:
from our plugin, we put customerData on every page which contains a user's email if they're logged in